### DISCUSSION TITLE
sample code to use message-ix-models

### DISCUSSION THREAD

**Discussion by zsalimian** [iiasa/message_ix#550] [MESSAGE modeling] [answered]
Dear Sir,

Is there any sample code to use message-ix-models like westeros?

Regards

**Comment by LauWien** [comment]
Dear @zsalimian,
Since this is neither about (a) a bug or error, nor (b) a requested feature in the software per se, I have transferred and converted this from an issue of the message-ix-models repository to a “Discussion” in the message_ix repository.

**Comment by LauWien** [comment]
Dear @zsalimian,
thank you for reaching out and your interest in message-ix-models! So far there is no sample code to run message-ix-models, but please see the [documentation](https://docs.messageix.org/projects/models/en/latest/) for further understanding.

**Comment by khaeru** [comment, marked as answer]
Duplicate of #531.
